pytorch_lightning/trainer/training_tricks.py

```diff
@@ -136,7 +136,9 @@ def scale_batch_size(self,
 
         """
         if not hasattr(model, batch_arg_name):
-            raise MisconfigurationException(f'Field {batch_arg_name} not found in `model.hparams`')
+            if not hasattr(model, 'hparams') or not hasattr(model.hparams, batch_arg_name):
+                raise MisconfigurationException(f'Neither of `model.batch_size` and'
+                                                f' `model.hparams.batch_size` found.')
```

Comment:
```suggestion
                raise MisconfigurationException(
                    'Neither of `model.batch_size` and `model.hparams.batch_size` found.'
                )
```
no need for `f` if there is no variable inside